cocos/animation/core/pose.ts

```diff
@@ -141,66 +141,66 @@ function copyTransformsWithFilter (target: TransformArray, source: Readonly<Tran
     }
 }
 
-const blendIntoTransformArrayAt = (() => {
+const blendIntoTransformArrayAt = ((): (target: TransformArray, source: Readonly<TransformArray>, alpha: number, transformIndex: number) => void => {
```

Comment:
`((): (target: TransformArray, source: Readonly<TransformArray>, alpha: number, transformIndex: number) => void => {`  ->. `((target: TransformArray, source: Readonly<TransformArray>, alpha: number, transformIndex: number) => void {`

Reply:
Please check other functions too.

Reply:
There is nothing wrong with the original way of writing, I think it is just more difficult to read, it may be easier to understand if it is changed to this:
```ts
type BlendIntoTransformArrayAtFunc = (target: TransformArray, source: Readonly<TransformArray>, alpha: number, transformIndex: number) => void;
const blendIntoTransformArrayAt = ((): BlendIntoTransformArrayAtFunc => {
//....
})();
```

Reply:
updated here https://github.com/cocos/cocos-engine/pull/15314/commits/4a30db77036fbde33de0e25d66807936e3a8b685